assimilation_code/modules/assimilation/algorithm_info_mod.f90

```diff
@@ -128,107 +290,198 @@ subroutine probit_dist_info(kind, is_state, is_inflation, dist_type, &
 ! In the long run, may not have to have separate controls for each of the input possibilities
 ! However, for now these are things that need to be explored for science understanding
 
-if(is_inflation) then
+! Fill arrays with possible dist_type strings and corresponding ints
+possible_dist_types(1) = 'NORMAL_DISTRIBUTION'
+possible_dist_types(2) = 'BOUNDED_NORMAL_RH_DISTRIBUTION'
+possible_dist_types(3) = 'GAMMA_DISTRIBUTION'
+possible_dist_types(4) = 'BETA_DISTRIBUTION'
+possible_dist_types(5) = 'LOG_NORMAL_DISTRIBUTION'
+possible_dist_types(6) = 'UNIFORM_DISTRIBUTION'
+possible_dist_types(7) = 'PARTICLE_FILTER_DISTRIBUTION'
+
+possible_dist_type_ints(1) = 1
+possible_dist_type_ints(2) = 2
+possible_dist_type_ints(3) = 3
+possible_dist_type_ints(4) = 4
+possible_dist_type_ints(5) = 5
+possible_dist_type_ints(6) = 6
+possible_dist_type_ints(7) = 7
+
+!use default values if qcf_table_filename is not in namelist
+if (.not. qcf_table_listed) then
+   dist_type = BOUNDED_NORMAL_RH_DISTRIBUTION
```

Comment:
This is just a question for my own understanding. Why is the default BOUNDED_NORMAL_RH_DISTRIBUTION
with no bounds, 
why not have the default NORMAL_DISTRIBUTION?

Reply:
@jlaucar there are two comments/questions on this pull request that you need to have a look at and answer. This is one of them.